testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/ResetPasswordTest.java

```diff
@@ -884,7 +885,7 @@ private void setPasswordPolicy(String policy) {
 
     @Test
     public void resetPasswordWithLengthPasswordPolicy() throws IOException, MessagingException {
-        setPasswordPolicy("length");
+        setPasswordPolicy(LengthPasswordPolicyProviderFactory.ID);
```

Comment:
Please remove this. Tests are by intent using a string and not a constant here.

Reply:
done